.env.example

```
@@ -1,5 +1,5 @@
 # MONGO_URI=
-MONGO_DB=
+MONGO_DB="mongodb://localhost:27017/myapp"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)